src/encoded/static/components/award.js

```diff
@@ -543,90 +1498,621 @@ ChartRenderer.propTypes = {
+
+    proposed_count: {
+        title: 'Proposed count',
+    },
 
-// Overall component to render the award charts
-const AwardCharts = (props) => {
-    const { award } = props;
 
+    deliverable_unit: {
+        title: 'Biosample',
+    },
+
+    contract_date: {
+        title: 'Contract date',
+    },
+};
+
+const MilestonesTable = (props) => {
+    const { award } = props;
     return (
-        <Panel>
-            <PanelHeading>
-                <h4>{award.pi && award.pi.lab ? <span>{award.pi.lab.title}</span> : <span>No PI indicated</span>}</h4>
-                <ProjectBadge award={award} addClasses="badge-heading" />
-            </PanelHeading>
-            <PanelBody>
-                <FetchedData>
-                    <Param name="experiments" url={`/search/?type=Experiment&award.name=${award.name}`} />
-                    <Param name="annotations" url={`/search/?type=Annotation&award.name=${award.name}`} />
-                    <ChartRenderer award={award} />
-                </FetchedData>
-            </PanelBody>
-        </Panel>
+        <SortTablePanel title="Milestones">
+            <SortTable list={award.milestones} columns={milestonesTableColumns} />
+        </SortTablePanel>
     );
 };
 
-AwardCharts.propTypes = {
-    award: PropTypes.object.isRequired, // Award represented by this chart
+MilestonesTable.propTypes = {
+    award: PropTypes.object.isRequired,
 };
 
+// Overall component to render the cumulative line chart
+const ExperimentDate = (props) => {
+    const { experiments, award } = props;
+    let dateReleasedArray = [];
+    let dateSubmittedArray = [];
+    const deduplicatedreleased = {};
+    const deduplicatedsubmitted = {};
+    let label = [];
+    let data = [];
+    let cumulativedatasetReleased = [];
+    let cumulativedatasetSubmitted = [];
+    let accumulatorreleased = 0;
+    let accumulatorsubmitted = 0;
+    let monthreleaseddiff = 0;
+    let monthsubmitteddiff = 0;
+    const fillreleasedDates = [];
+    const fillsubmittedDates = [];
+
+    // Search experiments for month_released and date_submitted in facets
+    if (experiments && experiments.facets && experiments.facets.length) {
+        const monthReleasedFacet = experiments.facets.find(facet => facet.field === 'month_released');
+        const dateSubmittedFacet = experiments.facets.find(facet => facet.field === 'date_submitted');
+        dateReleasedArray = (monthReleasedFacet && monthReleasedFacet.terms && monthReleasedFacet.terms.length) ? monthReleasedFacet.terms : [];
+        dateSubmittedArray = (dateSubmittedFacet && dateSubmittedFacet.terms && dateSubmittedFacet.terms.length) ? dateSubmittedFacet.terms : [];
+    }
+
+    // Use Moment to format arrays of submitted and released date
+    const standardreleasedTerms = dateReleasedArray.map((term) => {
+        const standardDate = moment(term.key, ['MMMM, YYYY', 'YYYY-MM']).format('YYYY-MM');
+        return { key: standardDate, doc_count: term.doc_count };
+    });
```

Comment:
This is a matter of taste, but I don’t think you should put the type of the variable in the variable name. Instead of `dateReleasedArray` I’d prefer releasedDates, for example.